perian/perian-core/index.js

```diff
@@ -8,31 +8,48 @@ import * as KeyPair from './src/commons/keyPair.js'
 import * as Network from './src/commons/network.js'
 import * as Base from './src/utils/base.js'
 import * as Crypto from './src/utils/crypto.js'
-import * as Plugin from './src/utils/plugin.js'
 
-const Utils = { Crypto, Base, Plugin }
+import * as bitcoin from '@perian/network-bitcoin'
+import * as bitcoincash from '@perian/network-bitcoincash'
+import * as bitcoincashsv from '@perian/network-bitcoincashsv'
+import * as dash from '@perian/network-dash'
+import * as digibyte from '@perian/network-digibyte'
+import * as dogecoin from '@perian/network-dogecoin'
+import * as eboost from '@perian/network-eboost'
+import * as feathercoin from '@perian/network-feathercoin'
+import * as bitcoingold from '@perian/network-bitcoingold'
+import * as groestlcoin from '@perian/network-groestlcoin'
+import * as litecoin from '@perian/network-litecoin'
+import * as qtum from '@perian/network-qtum'
+import * as smartcash from '@perian/network-smartcash'
+import * as uniformfiscalobject from '@perian/network-uniformfiscalobject'
+import * as vertcoin from '@perian/network-vertcoin'
+import * as zcoin from '@perian/network-zcoin'
+
+// $FlowFixMe
+Network.addNetworks({
+  bitcoin,
```

Comment:
This is great. We could actually put these imports and the Network.addNetwork calls into edge-currency-bitcoin. That way, edge-currency-bitcoin can control which plugins it jams in.

Reply:
Also, could `Network` be lower case? It's an instance, not a class or a type.